index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+  <main>
+    <div class="container">
+      <header>
+        <div class="logo-container">
+          <P>Emmanuel</P>
+            </div>
+              <div class="menu-btn">
+                <img src="./assets/menu-icon.png" alt="Menu Icon">
+              </div>
+      </header>
+      <div class="home">
+        <h1>Hey There.<br>I'm Emmanuel</h1>
+        <h2>I'm a software Developer</h2>
+        <p>I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.
+        </p>
+        <div class="social">
+          <div class="social-icons">
+            <div class="icon">
+              <img src="./assets/github-icon.png" alt="Github icon" class="icons">
+            </div>
+            <div class="icon">
+              <img src="./assets/linkedin-icon.png" alt="Linkedin icon" class="icons">
+            </div>
+            <div class="icon">
+              <img src="./assets/chill-icon.png" alt="Angelist Icon" class="icons">
+            </div>
+            <div class="icon">
+              <img src="./assets/medium-icon.png" alt="Medium Icon" class="icons">
+            </div>  
+          </div>
+          <div class="illustration">
+            <img src="./assets/illustration-header-mobile.svg" alt="Illustration">
+          </div>
```

Comment:
As you read in the [When to use HTML  tag and CSS background-image property](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html) article. It's recommended to use `background-image` instead of the HTML `img` tag when the image is only used for design. So make sure to use this opportunity for using the CSS `background-image` property.